rfcs/rfc-standalone-annual-report.md

```diff
@@ -0,0 +1,81 @@
+
+
+# Summary
+
+Create a standalone Annual Report filing and Beneficial Ownership registration to prove out the workflow and UX for capturing the data required for the Transparency Registry.
+
+# Motivation
+
+1. Creating a standalone AR filing that includes beneficial owner registration delivers on one of the key selling points made to the Digital Investment Board by Registries to undertake the registry assignment, namely, that the requirement for private corporations to make an annual report filing gave Registries the means to guarantee a mandated pathway to compliance for a large percentage of the more than 550,000 BC corporations.
+  
+2. The annual report standalone filing is a better place to start for collecting beneficial ownership information. For example, it would require less development work initially than starting with incorporation.
+  
+3. The sooner we can stand up a working version of the Register workflow, the sooner we can get it in front of users in the dev environment for UX research purposes. This is crucial because the OCM component of Registering beneficial owners will be extensive and challenging.
+  
+4. The sooner we can get data into the beneficial owner database (even just sample data) the sooner we can extend SOLR to beneficial owner database and include it along with director search to create the integrated Person Search.
+
+# Detailed design
+
+## Include Add a Beneficial Owner workflow
+The filing flow would be:
+The workflow is only relevant for BCA entities.
+1. Annual Report Date (this will be save to _lear_ or _colin_ as required)
+2. Significant Individual Assessment section
+3. UXA to determine:
+   a. Add Beneficial Owner Details: OR
+   b. Choose from Beneficial Ownership profile
+4. Possibly create a Beneficial Ownership profile
+
```

Comment:
Here's a sequence diagram to add in:
````suggestion
```sequenceDiagram
    actor Business Manager
    Business Manager->>+Annual Report: Need to file a businesses Annual Report
    Annual Report->>+Beneficial Ownership: Need to update Beneficial Owners
    Beneficial Ownership-->>-Annual Report: Beneficial Owners updated
    alt Business in Colin
      Annual Report->>Colin: File the AR Date
    else Business in Lear
          Annual Report->>Lear: File the AR Date
    end
    Annual Report-->>-Business Manager: Done!
```
````

Reply:
I think for COLIN, it would be a full filing entry that we'd need to add, with an event_ID that is in sequence with the rest of the COLIN database as well as the Last AR year field (which might be on the corporation table - and is used for calculating Not in good standing).

Reply:
We do that today for corps info we sync back to Colin. So that's not a new item to build